homeassistant/components/mill/strings.json

```
@@ -0,0 +1,21 @@
+{
+  "title": "Mill",
+  "config": {
+    "abort": {
+      "already_configured": "The Mill account is already configured."
```

Please use references. 
```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
```

---

FYI the latest version of the config flow scaffold script would put these in for you.